content/en/docs/ambient/architecture/_index.md

```diff
@@ -11,15 +11,22 @@ test: n/a
 
```

Comment:
We can leave the path as `/architecture` but can we do:


`description: A deep dive into the data plane of ambient mode.`

Reply:
This page will eventually be a header for the whole section, and not all of it will be about the data plane - some of it will be about the control plane implementation, I would think

Reply:
Ambient should not have any unique control plane docs (or we are doing something wrong) - those should be shared across sidecar and ambient.

Reply:
Yeah, the "labels you set" feels like they belong more in "usage" than "architecture" anyway tho - architecture is just "things you do not _absolutely_ need to know to use or follow our usage guides but which you or your compliance teams might want to have an understanding of" in my head.